cluster_support_bot/messages.py

```diff
@@ -52,18 +35,40 @@ def print_help(self, file=None):
         raise HelpRequest({'parser': self})
 
 
-@slack.RTMClient.run_on(event='message')
-def handle_message(**payload):
-    try:
-        # Exit if the message doesn't have ID
-        message_id = payload['data'].get('client_msg_id')
-        if not message_id:
-            return
-        asyncio.ensure_future(
-            _handle_message(msg_id=message_id, payload=payload),
-            loop=asyncio.get_event_loop())
-    except Exception as e:
-        logger.debug('uncaught Exception in handle_message: {}'.format(e))
+def handle_uuid_mention(text):
+    match = uuid_re.match(text)
+    if match:
+        uuid = match.groups()[0]
+        mention_counter.labels(uuid).inc()
+        return uuid
+
+
+def handle_help(payload, args=None, body=None, subparser=None):
+    web_client = payload['web_client']
+    channel = payload['data']['channel']
+    thread = payload['data'].get('thread_ts', payload['data']['ts'])
+    if not subparser:
+        subparser = parser
+    message = subparser.format_help()
+    return web_client.chat_postMessage(channel=channel, thread_ts=thread, text=message)
+
+
+def handle_parse_args_error(payload, error):
```

Comment:
>Can we break this down into more commits, so I can quickly skim over any moves/reformats, and more carefully review any actual functionality changes?

No functionality changes, most of this is done to allow proper package build/install along with "this needs to be moved out so that unit tests would work". Not sure if I can split this even more

Reply:
Broke this apart is several commits - __init__.py with main loop and flake8 fixes